### ISSUE TITLE
[BUG] ApkParser fails - Unexpected chunk

### ISSUE THREAD

**Issue by Stillhard** [ElderDrivers/EdXposed#736]
```
12-17 17:35:05.528  2769  2769 E EdXposed: ApkParser fails: pkg=com.google.android.gms, prc=com.google.android.gms
12-17 17:35:05.528  2769  2769 E EdXposed: com.jaredrummler.apkparser.exception.ParserException: Unexpected chunk Type:204
12-17 17:35:05.528  2769  2769 E EdXposed: 	at com.jaredrummler.apkparser.parser.ResourceTableParser.readChunkHeader(ResourceTableParser.java:204)
12-17 17:35:05.528  2769  2769 E EdXposed: 	at com.jaredrummler.apkparser.parser.ResourceTableParser.readPackage(ResourceTableParser.java:96)
12-17 17:35:05.528  2769  2769 E EdXposed: 	at com.jaredrummler.apkparser.parser.ResourceTableParser.parse(ResourceTableParser.java:67)
12-17 17:35:05.528  2769  2769 E EdXposed: 	at com.jaredrummler.apkparser.ApkParser.parseResourceTable(ApkParser.java:417)
12-17 17:35:05.528  2769  2769 E EdXposed: 	at com.jaredrummler.apkparser.ApkParser.transBinaryXml(ApkParser.java:269)
12-17 17:35:05.528  2769  2769 E EdXposed: 	at com.jaredrummler.apkparser.ApkParser.parseManifestXml(ApkParser.java:207)
12-17 17:35:05.528  2769  2769 E EdXposed: 	at com.jaredrummler.apkparser.ApkParser.parseApkMeta(ApkParser.java:199)
12-17 17:35:05.528  2769  2769 E EdXposed: 	at com.jaredrummler.apkparser.ApkParser.getApkMeta(ApkParser.java:150)
12-17 17:35:05.528  2769  2769 E EdXposed: 	at com.elderdrivers.riru.edxp._hooker.impl.HandleBindApp.beforeHookedMethod(HandleBindApp.java:66)
12-17 17:35:05.528  2769  2769 E EdXposed: 	at de.robv.android.xposed.XC_MethodHook.callBeforeHookedMethod(XC_MethodHook.java:51)
12-17 17:35:05.528  2769  2769 E EdXposed: 	at EdHooker_.hook(Unknown Source:94)
12-17 17:35:05.528  2769  2769 E EdXposed: 	at android.app.ActivityThread.access$1300(ActivityThread.java:219)
12-17 17:35:05.528  2769  2769 E EdXposed: 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1859)
12-17 17:35:05.528  2769  2769 E EdXposed: 	at android.os.Handler.dispatchMessage(Handler.java:107)
12-17 17:35:05.528  2769  2769 E EdXposed: 	at android.os.Looper.loop(Looper.java:214)
12-17 17:35:05.528  2769  2769 E EdXposed: 	at android.app.ActivityThread.main(ActivityThread.java:7356)
12-17 17:35:05.528  2769  2769 E EdXposed: 	at java.lang.reflect.Method.invoke(Native Method)
12-17 17:35:05.528  2769  2769 E EdXposed: 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:491)
12-17 17:35:05.528  2769  2769 E EdXposed: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:925)
```

Android version: 10
Android sdk: 29
EdXposed version: v0.5.1.0 (YAHFA)
EdXposed api: 93
Riru version: v23.0 (50)
Riru api: 10
Magisk: 21.1 (21100)

**Comment by yujincheng08** [comment]
jaredrummler/APKParser#17

**Comment by yujincheng08** [comment]
Fixed in 43c4c92365d401b22eb1ef14a928ff53822404d4
